src/main/kotlin/no/nav/familie/ks/sak/integrasjon/familieintegrasjon/IntegrasjonClient.kt

```diff
@@ -524,6 +524,21 @@ class IntegrasjonClient(
         }
     }
 
+    fun sjekkErEgenAnsattBulk(personIdenter: List<String>): Map<String, Boolean> {
```

Comment:
Trenger vi suffixen "Bulk"? Føler det kommer godt fram når man sender inn en liste/set av strings.

Litt flisespikkeri med kan vurdere `set` over `liste` da jeg det ikke er så mye vits å sjekke en ident flere ganger?

```suggestion
    fun sjekkErEgenAnsatt(personIdenter: List<String>): Map<String, Boolean> {
```

Reply:
Valgte det fordi endepunktet den kaller mot er /bulk. Det finnes et endepunkt uten /bulk path for enkeltpersoner. Med tanke på at man som oftest også skal ha med relasjoner tviler jeg på at det vil brukes for enkeltpersoner. Endrer navn og param ✅